src/linpde_gp/randprocs/covfuncs/linfuncops/diffops/_registry.py

```diff
@@ -29,121 +30,133 @@ def _(self, k: covfuncs.JaxCovarianceFunctionMixin, /, *, argnum=0):
         )
 
 
-########################################################################################
-# Partial Derivative ###################################################################
-########################################################################################
-
-
-@diffops.Derivative.__call__.register  # pylint: disable=no-member
-def _(self, k: pn_covfuncs.Matern, /, *, argnum: int = 0):
+@diffops.PartialDerivative.__call__.register  # pylint: disable=no-member
+@diffops.LinearDifferentialOperator.__call__.register  # pylint: disable=no-member
```

Comment:
```suggestion
@diffops.LinearDifferentialOperator.__call__.register  # pylint: disable=no-member
```

Reply:
From what I've tried out, we actually need to register this for `PartialDerivative` as well. Otherwise it won't get called for `PartialDerivative` instances, even though it is a `LinearDifferentialOperator`. I think that's because `PartialDerivative` doesn't fallback to `LinearDifferentialOperator`'s call (which it can't, because otherwise we would get a cyclic dependency).

Reply:
As discussed offline, this might be mitigated by handling the `PartialDerivative` fallback explicitly in `LinearDifferentialOperator.__call__`.